bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/api/GoEStatusResponseV2DTO.java

```diff
@@ -24,30 +26,30 @@ public class GoEStatusResponseV2DTO extends GoEStatusResponseBaseDTO {
     @SerializedName("mod")
     public String version;
 
-    @SerializedName("psm")
+    @SerializedName(PSM)
     public Integer phases;
 
-    @SerializedName("trx")
+    @SerializedName(TRX)
     public Integer transaction;
 
-    @SerializedName("alw")
+    @SerializedName(ALW)
```

Comment:
What is the purpose of changing all these annotations from a string literal to a constant ? 
On first sight it seems to create complexity and a file with 1300+ lines without a clear goal. Or what am i missing?